index.html

```diff
@@ -114,9 +114,7 @@ <h2>
         <li>Let <var>valid pattern</var> be the result of passing
         <var>pattern</var> to <a>validate and normalize</a>.
         </li>
-        <li>If the result of running the steps to <a href=
-        "https://w3c.github.io/page-visibility/#dfn-steps-to-determine-the-visibility-state">
-          determine the visibility state</a> [[!PAGE-VISIBILITY-2]] is not
+        <li>If the <a>document</a>'s {{Document/visibilityState}} is not 
```

Comment:
Link to the "visibility state" concept rather than the script-exposed `visibilityState` attribute getter steps.

```suggestion
        <li>If the <a>document</a>'s [=Document/visibility state=] is not 
```